packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -369,19 +367,19 @@ class _RenderCupertinoPickerSemantics extends RenderProxyBox {
   int _currentIndex = 0;
 
   void _handleIncrease() {
-    controller.jumpToItem(_currentIndex + 1);
+    controller!.jumpToItem(_currentIndex + 1);
```

Comment:
Same here and below where controller is used.

Reply:
Changed.